### ISSUE TITLE
STUB: Plan UX testing for supplemental pages and attachments features

### ISSUE THREAD

**Issue by jimmoffet** [GSA-TTS/atj-platform#309]
## Overview

As a _, I would like _, so that I can _.

## Context

*Optional: Any reference material or thoughts we may need for later reference, or assumptions of prior or future work that's out of scope for this story.*

- [ ]

## Acceptance Criteria

*Required outcomes of the story*

- [ ]

## Research Questions

- *Optional: Any initial questions for research*

## Tasks

*Research, design, and engineering work needed to complete the story.*

- [ ]

## Definition of done

The "definition of done" ensures our quality standards are met with each bit of user-facing behavior we add. Everything that can be done incrementally should be done incrementally, while the context and details are fresh. If it’s inefficient or “hard” to do so, the team should figure out why and add OPEX/DEVEX backlog items to make it easier and more efficient.

- [ ] Behavior
  - [ ] Acceptance criteria met
  - [ ] Implementation matches design decisions
- [ ] Documentation
  - [ ] ADRs (`/documents/adr` folder)
  - [ ] Relevant `README.md`(s)
- [ ] Code quality
  - [ ] Code refactored for clarity and no design/technical debt
  - [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies; dependency rule followed
  - [ ] Code is reviewed by team member
  - [ ] Code quality checks passed
- [ ] Security and privacy
  - [ ] Automated security and privacy gates passed
- [ ] Testing tasks completed
  - [ ] Automated tests pass
  - [ ] Unit test coverage of our code >= 90%
- [ ] Build and deploy
  - [ ] Build process updated
  - [ ] API(s) are versioned
  - [ ] Feature toggles created and/or deleted. Document the feature toggle
  - [ ] Source code is merged to the main branch

## Decisions

- *Optional: Any decisions we've made while working on this story*

**Comment by JennyRichards-Flexion** [comment]
Ethan's big question - when someone attaches something, what happens to it? Does it go through an S3 bucket, render to a canvas and spat out into PDF? We need to have a conversation about what happens when someone uploads. 
Jim: Serialize it, save to Postgres - then render to canvas and attach to the PDF object during the generate process? There is an HTML canvas in PDF lib - it should be fine.

**Comment by JennyRichards-Flexion** [comment]
RE: Blocker in Slack:
Jim: Merging of prototype of the component is blocked on another merge, but should be resolved this week. Should be able to fetch the staging build for the repeater PR. Should be enough for you to see what's in there.

**Comment by JennyRichards-Flexion** [comment]
Will start on this next week.

**Comment by nichole-aquino** [comment]
Reviewed [research plan](https://docs.google.com/document/d/10VmpqjnMalnMR-2Nhogwa7KiQXokrUBxvK4IhEXZ_n4/edit?tab=t.0) with the team. Preparing for testing week in Sprint 12.

**Comment by nichole-aquino** [comment]
[Summary of findings](https://docs.google.com/document/d/12lWxtAJ17WwCqCM1ehwF2SOGyX_cX0bsBCFnDKStzi8/edit?tab=t.0#heading=h.tjmzj6h5r8u9) shared with the team on 12/6.
